app/views/recipes/index.html.erb

```diff
@@ -1,14 +1,19 @@
-<p style="color: green"><%= notice %></p>
+<div class="column">
+  <h3 class="title has-text-white-ter has-text-right">
+    My recipes
+  </h3>
+  <hr class="login-hr">
+  <p class="subtitle has-text-white-ter has-text-right">
+    <%= "I have already created #{@recipes.length} #{@recipes.length == 1 ? "recipe" : "recipe".pluralize}!" %>
```

Comment:
You can rely on I18n for [pluralization](https://guides.rubyonrails.org/i18n.html#pluralization)